lib/fakes3/version.rb

```diff
@@ -1,3 +1,3 @@
 module FakeS3
-  VERSION = "1.2.1"
+  VERSION = "1.2.3"
```

Comment:
Ah, I forgot about this version field... I have been tagging by hand with annotated tags.